### ISSUE TITLE
Listen on IPv6 addresses and support AAAA records

### ISSUE THREAD

**Issue by fhunleth** [pcmarks/mdns_lite#39]
IPv6 has one feature that's really nice and that's the ability to start with an IPv6 address already assigned and not need a DHCP server to work. IPv4 can do a similar thing with Link Local addressing, but that usually only works after DHCP fails. 

It looks like the following changes are required for IPv6 support:

1. Remove the filter that prevents responders from being started on IPv6 addresses
2. Change the call to `gen_udp` to listen on the mDNS IPv6 address
3. Use AAAA records instead of A ones

**Comment by pcmarks** [comment]
Would we support IPv4 a welll as IPv6? Would the user be able to choose between two?

**Comment by fhunleth** [comment]
Yes, I would expect both to be working at the same time. I don’t think it’s necessary to support turning one or the other off.
